src/server/ua_server_binary.c

```diff
@@ -1760,6 +1764,7 @@ UA_BinaryProtocolManager_start(UA_Server *server,
     if(!haveServerSocket) {
         UA_LOG_ERROR(config->logging, UA_LOGCATEGORY_SERVER,
                      "The server has no server socket");
+        return UA_STATUSCODE_BADINTERNALERROR;
     }
 
     /* Update the application description to include the server urls for
```

Comment:
@jpfr How do we deal with this?
Because we use only the wildcard option by default, no URL is added to the DiscoveryUrl list.

Reply:
Is this a problem?
I think we mirror back the DiscoveryUrl used to open the connection.
When somebody calls GetEndpoints...